### PR TITLE
Game Base Fix hover effect

### PR DESCRIPTION
**Pull request by AaryanSingh0017**
I have put the code I hope its correct.

### CODE REVIEW

File: gameBase/src/css/Hover.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*Use media hover with media pointer will help you resolve this issue. Tested on chrome Web and android mobile.*/
+
+@media (hover: hover) and (pointer: fine) {
```
Comment (GeraAlcantara) [inline gameBase/src/css/Hover.css:3]: you just need the query to @media (hover: hover) remove the and (pointer;fine) part 
and apply this concept to the style css file I'll tri to point the line where you need to aplly the change

Comment (GeraAlcantara) [reply gameBase/src/css/Hover.css:3]: gameBase/src/css/styles.css
line 39 
apply the media query rule there

Comment (GeraAlcantara) [reply gameBase/src/css/Hover.css:3]: and delete this file

**Review by GeraAlcantara — CHANGES_REQUESTED** [review]

**Comment by GeraAlcantara** [comment]
issue was fix on PR #61

**Comment by AaryanSingh0017** [comment]
okk

**Comment by AaryanSingh0017** [comment]
Plz remove the invalid tag its showing in Hacktoberfest, i Just made a minor mistake , I am new to open source and you can assign new task to me.

**Pull request closed without merging**